tests/serialization/test_dag_serialization.py

```diff
@@ -523,7 +523,7 @@ def validate_deserialized_task(
         if isinstance(task.params, ParamsDict):
             assert serialized_task.params.dump() == task.params.dump()
 
-        # Check that for Deserialised task, task.subdag is None for all other Operators
+        # Check that for Deserialized task, task.subdag is None for all other Operators
```

Comment:
Both the spellings are correct: serialization (US spelling) or serialisation (UK spelling)

Reply:
Should we standardise (or rather standardize :D)  the US spelling ? I think this is what we mostly have in our spelling_wordlist and docs :).

Reply:
@kaxil  I searched across the repository:
- `deserialis*` - 4 matches
- `deserializ*` - 100+ matches
It's better to choose one mostly used spelling.

Reply:
I think there is no need to standardi(s/z)e this tbh :D . Both should be accepted - we gain nothing by this standardization.